src/server/services/CloudmersiveVirusScanService.ts

```diff
@@ -0,0 +1,40 @@
+import fileUpload from 'express-fileupload'
+import { injectable } from 'inversify'
+import { ApiClient, ScanApi } from 'cloudmersive-virus-api-client'
+import { VirusScanServiceInterface } from './interfaces/VirusScanServiceInterface'
+import { cloudmersiveKey, logger } from '../config'
+
+if (cloudmersiveKey) {
+  const client = ApiClient.instance
+  const ApiKey = client.authentications.Apikey
+  ApiKey.apiKey = cloudmersiveKey
+}
+const api = new ScanApi()
+
+@injectable()
+export class CloudmersiveVirusScanService implements VirusScanServiceInterface {
+  private static scanFilePromise: (file: Buffer) => Promise<boolean> = (file) =>
+    new Promise((res, rej) => {
+      api.scanFile(file, (err, data) => {
```

Comment:
what's wrong with just using util.promisify?

Reply:
doesnt work well with typescript :( the `scanFilePromise` resultant type interpreted was a mess